lib/screens/edit_product_screen.dart

```diff
@@ -86,6 +98,49 @@ class _EditProductScreenState extends State<EditProductScreen> {
     );
   }
 
+  String validatorImageUrl(String value) {
+    if (value.isEmpty) {
+      return 'Please enter an image URL';
+    }
+    if (value.startsWith('http') && !value.startsWith('https')) {
+      return 'Please enter a valid URL.';
+    }
+    if (!value.endsWith('.png') && !value.endsWith('jpg') && !value.endsWith('.jpeg')) {
+      return 'Please enter a valid image URL.';
+    }
+    return null;
```

Comment:
try:

`return value.isEmpty ? 'Please enter an image URL' : !value.startsWith('https') ? 'Please enter a valid URL.' : !value.endsWith('.png') || !value.endsWith('jpg') || !value.endsWith('.jpeg') ? 'Please enter a valid image URL.' : null`

Reply:
![image](https://user-images.githubusercontent.com/74462760/104462755-ed30b580-55eb-11eb-9bdd-842be4960fd4.png)

I tried implementing it but even if the URL starts with https and ends with jpg, it still shows that there is an error in the URL. Please see below.

![image](https://user-images.githubusercontent.com/74462760/104462966-26692580-55ec-11eb-8a4b-444d027f7d3b.png)

Reply:
@MapleSyyrup can you send here the value of image url you are trying to display?

Reply:
https://upload.wikimedia.org/wikipedia/commons/thumb/1/14/Cast-Iron-Pan.jpg/1024px-Cast-Iron-Pan.jpg

Reply:
I see. Then just go with the original implementation for now.